client-ts/Microsoft.AspNetCore.SignalR.Client.TS/Microsoft.AspNetCore.SignalR.Client.TS.csproj

```diff
@@ -13,7 +13,6 @@
 
   <ItemGroup>
     <Inputs Include="*.ts;" />
-    <Inputs Remove="Observable.ts" />
```

Comment:
Yea, it *really* can't handle the whole "using tooling" and globbing support....